ui/pages/confirm-add-suggested-token/confirm-add-suggested-token.stories.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable react/prop-types */
+import React, { useEffect } from 'react';
+import { text } from '@storybook/addon-knobs';
+import { store } from '../../../.storybook/preview';
+import { suggestedTokens } from '../../../.storybook/initial-states/approval-screens/add-suggested-token';
+import { updateMetamaskState } from '../../store/actions';
+import ConfirmAddSuggestedToken from '.';
+
+export default {
+  title: 'Confirmation Screens',
+};
+
+const PageSet = ({ children }) => {
+  const symbol = text('symbol', 'DAI');
+  const image = text(
+    'Icon URL',
+    'https://assets.coingecko.com/coins/images/9956/small/dai-multi-collateral-mcd.png?1574218774',
```

Comment:
Should we save this to a static directory for the storybook to use? Then we aren't hot linking to the resource

Reply:
yeah good point